ranger.py

```diff
@@ -125,14 +153,14 @@ def __init__(self, level):
         else:
             attacks = [weapon]
         base_feats.append(EquipWeapon(weapon))
-        base_feats.append(RangerAction(attacks=attacks))
+        base_feats.append(RangerAction(attacks=lambda: attacks, summon_fey=4))
         base_feats.append(Spellcasting(level, half=True))
         if level >= 20:
-            base_feats.append(HuntersMarkFeat(10, True))
+            base_feats.append(HuntersMarkFeat(10, True, ))
```

Comment:
Nit: Is there a reason for the extra commas?

Reply:
I think the formatter started adding those automatically